src/corner_kick/src/containers/mainUI/GlobalStyle.tsx

```diff
@@ -0,0 +1,186 @@
+/**
+ * This file defines the main styling of the application and connects the
+ * theme to all styled-components instances
```

Comment:
Quick note that this is where the overall panel "layout" is defined would be good here.